cluster/kube-down.sh

```diff
@@ -28,6 +28,18 @@ echo "Bringing down cluster using provider: $KUBERNETES_PROVIDER"
 
 verify-prereqs
 teardown-monitoring
+
+# If logging with Fluentd to Elasticsearch was enabled then
+# delete the pods and service and remove the firewall rule
+# for accessing Elasticsearch and Kibana.
+if [[ "${ENABLE_NODE_LOGGING-}" == "true" ]]; then
```

Comment:
Agreed, and as I said, I'll make the change. Although I am not sure if that's an approved use of the term *polymorphic* :-) Perhaps that's because I assume "parametric" in front of the word but others assume "ad hoc".